ruby_event_store/lib/ruby_event_store/specification.rb

```diff
@@ -33,14 +33,36 @@ def from(start)
     # Limits the query to events before or after another event.
     # {http://railseventstore.org/docs/read/ Find out more}.
     #
-    # @param start [String] id of event to start reading from.
+    # @param stop [String] id of event to start reading from.
     # @return [Specification]
     def to(stop)
       raise InvalidPageStop if stop.nil? || stop.empty?
       raise EventNotFound.new(stop) unless reader.has_event?(stop)
       Specification.new(reader, result.dup { |r| r.stop = stop })
     end
 
+    # Limits the query to events before or after another event.
+    # {http://railseventstore.org/docs/read/ Find out more}.
+    #
+    # @param date [Date]
+    # @param equals [Boolean] whether you want to include events on that date
+    # @return [Specification]
+    def older_than(date, equals = false)
+      raise InvalidPageStart if date.nil? || date.empty?
+      Specification.new(reader, result.dup { |r| r.older_than = [date, equals] })
+    end
+
+    # Limits the query to events before or after another event.
+    # {http://railseventstore.org/docs/read/ Find out more}.
+    #
+    # @param date [Date]
+    # @param equals [Boolean] whether you want to include events on that date
+    # @return [Specification]
+    def newer_than(date, equals = false)
+      raise InvalidPageStop if date.nil? || date.empty?
```

Comment:
The same here. Misleading error. Probably we need a new one or just use `ArgumentErrror`

Reply:
yep, I just wanted to prototype an initial idea so I did it by copy/pasting `from/to` methods. Will translate this exceptions to meaningful ones.